src/electron/views/screenshot/screenshot-container.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import { ScreenshotData } from 'common/types/store-data/unified-data-interface';
+import { BoundingRectangle } from 'electron/platform/android/scan-results';
+import { HighlightBox } from 'electron/views/screenshot/highlight-box';
+import { Screenshot } from 'electron/views/screenshot/screenshot';
+import { screenshotContainer } from 'electron/views/screenshot/screenshot-container.scss';
+import { isEmpty } from 'lodash';
+import * as React from 'react';
+
+export interface ScreenshotContainerProps {
+    screenshotData?: ScreenshotData;
+    highlightBoxRectangles?: BoundingRectangle[];
```

Comment:
`highlightBoxRectangles` can be non-optional (it's not optional in the view model, so it's the view model provider's job to normalize it as an empty list)

Reply:
Made highlightBoxRectangles prop required and updated screenshot-container.test.tsx accordingly